integration-tests/contract-call-with-xtz-amount.spec.ts

```diff
@@ -84,7 +86,9 @@ CONFIGS().forEach(({ lib, rpc, setup, protocol }) => {
         balance = await Tezos.tz.getBalance(contract.address);
         expect(balance.toString()).toEqual('1000001');
         done();
-      }
+      } catch (ex) {
+        console.error(ex);
```

Comment:
In case of an error, the test should fail instead of logging the error.

Reply:
sorry forgot to remove the added debugging console log line.  Removed.